code/modules/mob/living/silicon/ai/ai_mob.dm

```diff
@@ -705,15 +705,15 @@ GLOBAL_LIST_INIT(ai_verbs_default, list(
 
 	if(href_list["track"])
 		var/mob/living/target = locate(href_list["track"]) in GLOB.mob_list
-		if(target && target.can_track())
+		if(target && istype(target) && target.can_track())
```

Comment:
GLOB.mob_list can contain non-living mobs, like observers. If the AI gets the wrong locate, and tries to track an observer, it will say "Target is not on or near any active cameras on the station." even if the living mob they selected is trackable. Can you convert `GLOB.mob_list` to `GLOB.mob_living_list` to err on the side of caution?

Reply:
That will only happen if the mob was deleted, AIs can't track deleted mobs.
I'm very hesitant to have this check the living list because it pretty often breaks, although I probably should just fix that.